app/models/document.rb

```diff
@@ -59,13 +59,16 @@ def doc_expire_not_in_the_past
 #
 # Table name: documents
 #
-#  id         :bigint           not null, primary key
-#  user_id    :bigint           unsigned, not null
-#  upload     :string(255)
-#  doc_type   :string(255)
-#  doc_number :string(255)
-#  doc_expire :date
-#  metadata   :text(65535)
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
+#  id            :bigint           not null, primary key
+#  user_id       :bigint           unsigned, not null
+#  upload        :string(255)
+#  doc_type      :string(255)
+#  doc_number    :string(255)
+#  doc_expire    :date
+#  identificator :string(255)
+#  metadata      :text(65535)
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  doc_issue     :date
```

Comment:
How it relates to this PR?

Reply:
We havent updated annotates for a long time (its not automatic in barong). As soon as i have added new model - i've re-runed an annotations. You can see there is no difference in schema but only in annotations in models)

Reply:
I've thought about it)